samples/client/petstore/go/go-petstore/model_map_test_.go

```diff
@@ -11,7 +11,15 @@ package petstore
 
 type MapTest struct {
 	MapMapOfString map[string]map[string]string `json:"map_map_of_string,omitempty"`
-	MapOfEnumString map[string]string `json:"map_of_enum_string,omitempty"`
+	MapOfEnumString MapTestMapOfEnumString `json:"map_of_enum_string,omitempty"`
```

Comment:
This isn't quite right, I think it needs to be
```go
ArrayEnum map[string] MapTestMapOfEnumString```
on this line and
```go
type MapTestMapOfEnumString string```
one line 19 below

I'll work on a fix...

Reply:
Fixed